### ISSUE TITLE
Can't browse the public local feed of another instance while logged in (but can while logged out)

### ISSUE THREAD

**Issue by jaredly** [elk-zone/elk#972]
If I'm logged out, I can scroll through another server's local feed (like https://main.elk.zone/mas.to/public/local), but when I'm logged in to my mastodon.social account, it stops working, and just redirects to `/home`. I'd love to be able to view other instances `local` feeds 🙏

**Comment by KaKi87** [comment]
Would you say that you're making the same request as mastodon/mastodon#6942 ?

**Comment by ZMYaro** [comment]
Would you consider renaming this bug to something not specific to mas.to?  It didn't surface in a search before I filed #1646.

**Comment by ZMYaro** [comment]
> Would you say that you're making the same request as [mastodon/mastodon#6942](https://github.com/mastodon/mastodon/issues/6942) ?
I don't think so.  This isn't about being able to browse another instance from within one instance; it is about adding the functionality to Elk.  If we aren't planning to add the functionality to Elk, then we should at least change Elk's URL structure.

**Comment by KaKi87** [comment]
> This isn't about being able to browse another instance from within one instance

Well, the notion of "within" only exists in the issue I referenced because the frontend is attached to the instance, but that's not Elk's case.

What I mean is, is OP asking for the ability to browse the local feed of any instance from Elk, as opposed to opening said instance's frontend to do it and then copy/pasting profile/post URLs to Elk's search input in order to interact with those.

**Comment by boehs** [comment]
> Would you consider renaming this bug to something not specific to mas.to? It didn't surface in a search before I filed https://github.com/elk-zone/elk/issues/1646.

I'm sorry, I do not have this ability
